### ISSUE TITLE
`mongotail [db] --follow` exits after 1 second

### ISSUE THREAD

**Issue by kevinburke** [mrsarm/mongotail#24]
I installed the tool and am running `mongotail [db] --follow`. I expect that the command will block until I kill it with `ctrl+c`. Instead, it exits after 1 second, without printing anything.

It would be good if it warned that you needed to run with `--level=2`, if `--level=2` was the default mode, or otherwise displayed a warning or error, instead of exiting without displaying any messages.

**Comment by kevinburke** [comment]
Running `mongotail accounts --level=1 -f` sets the level and then exits. I expect that it would set the level, and then start following the database.

All of these make it difficult to script mongotail, or to explain how to others to use it; I'm looking for something that can immediately start tailing the database logs.

**Comment by mrsarm** [comment]
@kevinburke You are right. It would be a good improvements, and the last behavior is a bug actually. I'll try to implement the changes, but it will not be soon. Patchs are welcome :)

**Comment by ghost** [comment]
It always just exits for me.  I can't get it to work.

**Comment by kevinburke** [comment]
FWIW, I implemented a tool that has the behavior I want, https://github.com/kevinburke/read-mongo-logs

**Comment by mrsarm** [comment]
@errorx666 did you solve the issue?

Did you activate first the profiler with the `-l` option?

Mongotail has the same syntax to connect to a database that the command `mongo` provided by any MongoDB distribution, did you try to connect within the same console to the database with the same params but using the mongo command?

For instance, if you are executing `mongotail myhost/mydatabase`, you can try the same but using `mongo` command like this: `mongo myhost/mydatabase`. If you cannot connect to the database using this command, it's a connection issue. Try also connect with mongotail using the `--verbose` option.

**Comment by serkandemirel0420** [comment]
@errorx666  indeed, it did not work for me either. verbose did not do the trick either.  connection confirmed.

**Comment by mrsarm** [comment]
@errorx666 ,

Lets say you try to connect with your database with:

```
$ mongotail hostname/mydb -l 2
$ mongotail hostname/mydb -f
```

Can you try in the same machine execute:

```
$ mongo hostname/mydb
```

(note that you have to make the test in the same machine, using the same connection string, in the example `hostname/mydb`, but using the MongoDB `mongo` CLI)
